packages/docs/docs/docs/deploying.md

```diff
@@ -26,49 +24,55 @@ Use `npm` to create a `package.json` file:
 npm init
 ```
 
-This command will ask you for details about the project. For this very basic
-guide you can just press enter to accept the default values of each field.
+This command will ask you for details about the project. For this getting started project, just press **Enter** to accept the default values for each field.
 
-Then, install ZeppelinOS in your project folder by running:
+Then install ZeppelinOS in your project folder by running:
 
 ```console
-npm install zos
+npm install zos@2.3.0-rc.2
 ```
 
-Run `npx zos --help` to get a list of all the ZeppelinOS commands available. At any time
-during these guides, when we introduce a new command, run
-`npx zos <command-name> --help` to get more information about that command and
-its arguments.
+Run `npx zos --version` to output the current ZeppelinOS version.
 
-Now, we can initialize the ZeppelinOS project:
+Run `npx zos --help` to output a list of all ZeppelinOS commands available. At any time
+during this getting started tutorial, you can run `npx zos <command-name> --help` to get detailed information about that command and its arguments.
+
+The contract in this tutorial imports a contract from the ZeppelinOS library package, which includes contracts to help with the development of ZeppelinOS compatible smart contracts. Install the `zos-lib` package as follows:
+
+```console
+npm install zos-lib@2.3.0-rc.2
+```
+
+For this tutorial, you import `Initializable.sol` so your contract can use the `initializer` modifier, and thus prevent the `initialize` function from being called more than once.
```

Comment:
Readers don't know what is an initializer at this point. We may need to explain this later, or add a more complete explanation here.